analysis_model/squat/squat_real_tts.py

```
@@ -0,0 +1,617 @@
+import cv2
+import mediapipe as mp
+import numpy as np
+import os
+import time
+import threading
+import queue
+import subprocess
+from typing import List, Dict, Tuple, Optional
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import

(F401)